### PR TITLE
Covariances and parameters

### PR DESCRIPTION
**Pull request by samkys**
If you are interested, this one parameterized the launch file rather than having all parameters passed in it calls a .yaml file. Also unpublished covariances are set via parameters and published on respective topics. RPY uncertainties are also published.

Note: Please verify RPY uncertainties are correct if you merge.

### CODE REVIEW

**Comment by samkys** [comment]
Realized I sent pull request too early. Leave open a little longer as I need to clean up the .yaml file.

**Comment by samkys** [comment]
Updated params file per your comment. Check the code now.

**Comment by dawonn** [comment]
Hey, great work, and thank you for the PR. Just a couple small cleanup items and I'll merge.

**Comment by samkys** [comment]
Sorry to keep bugging you but one last update. I updated the async rate because with the previous async rate of 100 and the baud rate at 115200 the data rate was too slow for all of the info requested from the IMU should work now.

File: vnproglib-1.1/cpp/src/sensors.cpp [hunk 1]
```diff
@@ -460,8 +460,10 @@ struct VnSensor::Impl
 			groups |= 0x0004;
 		if (fields.gpsField)
 			groups |= 0x0008;
-		if (fields.attitudeField)
+		if (fields.attitudeField){
 			groups |= 0x0010;
+			printf("There is an Attitude Field\n");
```
[the remark this answers is not in the document]
Comment (samkys) [reply vnproglib-1.1/cpp/src/sensors.cpp]: Feel free and remove it. I had some help on this and didn't catch where it was placed. Sorry for not catching the backups. Remove any of them as well.

**Comment by dawonn** [comment]
Can you update the PR for me? I'm on mobile and don't have my dev machine
readily accessible.

**Comment by samkys** [comment]
Take a look at it now, I replaced the library with the original from your fork so it should be good.

**Pull request merged**